### DISCUSSION TITLE
How to disable redirect on browser when i have an error on my code on nextjs?

### DISCUSSION THREAD

**Discussion by ghost** [vercel/next.js#50640] [Help] [answered]
### Summary

Hello I try next js today but it's such a hassle because already when i have an error in my code next js switch immediately on my browser for show error but I just haven’t finished writing my code. We can tell a nextjs not to display the error on the browser directly

I would like it to react and not open the browser when I have an error. I tried to disable eslint and typescript in strict mode but I did not succeed and I found nothing on the internet

Next js is good but just this system makes me want to stop using it

Thank you for your help

**Comment by icyJoseph** [comment, marked as answer]
Hi,

Is this happening on Safari, when Safari is the default browser? It is such a weird bug... It's more like, Safari steals the focus, and I can't figure out why! In the mean time, please use other browser. I am still investigating what's going on...

**Reply by ghost** [reply]
Ok yes it's safari, i don't have the problem on chrome, thank you !

**Reply by icyJoseph** [reply]
It is so strange, first time I saw it I could not believe it. I've tried to see if there's some kind of A11Y reason, like `alert` role or aria-live set to assertive, but nothing... so far...

**Reply by icyJoseph** [reply]
And please don't forget to mark as answered too 😄
